frontend/src/app/page.tsx

```diff
@@ -1,13 +1,275 @@
+import Image from 'next/image';
+import { GoDotFill } from 'react-icons/go';
+
+const faqs = [
+  {
+    title: 'What is Youcompare ?',
+    content:
+      'Youcompare is a platform that helps you find the best tools for your business. We provide comprehensive comparisons of cloud services, databases, CMS platforms, and project management tools, so you can make informed decisions.',
+  },
+  {
+    title: 'How does the platform assist small business ?',
+    content:
+      'Our platform is designed to help small businesses find the right tools for their needs. We provide detailed comparisons of various services, so you can easily find the best fit for your business.',
+  },
+  {
+    title: 'Is the platform free for users ?',
+    content:
+      'Yes, our platform is completely free for users. You can explore and compare different tools without any cost or commitment.',
+  },
+  {
+    title: 'Is the platform up to date with prices ?',
+    content:
+      'Yes, we regularly update our platform with the latest prices and features of various tools.',
+  },
+];
+
+const allFeatures = [
+  {
+    title: 'Comprehensive Comparisons',
+    description:
+      'Easily compare cloud services, databases, CMS platforms, and project management tools side by side. Get all the key details you need in one place.',
+  },
+  {
+    title: 'Price Tracking',
+    description:
+      'Stay updated with real-time pricing changes and exclusive deals, so you always get the best value for your investment.',
+  },
+  {
+    title: 'Customizable Filters',
+    description:
+      'Narrow down your search based on pricing, features, scalability, and user reviews to find the solution that fits your unique requirements.',
+  },
+  {
+    title: 'Real-Time Updates',
+    description:
+      'Stay informed with live pricing and feature updates so you always have the latest information at your fingertips.',
+  },
+  {
+    title: 'User-Friendly Interface',
+    description:
+      'Designed with simplicity in mind, our platform makes complex comparisons effortless, whether youre a beginner or a pro.',
+  },
+  {
+    title: 'Comprehensive Comparisons',
+    description:
+      'Easily compare cloud services, databases, CMS platforms, and project management tools side by side. Get all the key details you need in one place.',
+  },
+];
 
 const HomePage = () => {
   return (
-    <div className="flex  items-center justify-center h-full">
-      <h1 className="text-4xl font-bold text-gray-700">
-        Select a Category from the Drawer
+    <ImageBackground>
+      <LandingPageHeader />
+      <HeroSection />
+      <div className="blur-[10px] -mt-12 bg-white p-12" />
+      <FeaturesIntroSection id="features" />
+      <div className="bg-white py-5 font-roboto px-4">
+        <AllFeaturesSection />
+        <FaQSection id="faqs" />
+      </div>
+      <Footer />
+    </ImageBackground>
+  );
+};
+
+const HeroSection = () => {
```

Comment:
I would put this components here into the Components directory and import them here to keep this page.tsx clean and easy to read.

move them out into component directory and export const them

Reply:
resolved